test/test_config.hh.in

```diff
@@ -25,35 +25,9 @@
 
 #define IGN_CONFIG_PATH "@CMAKE_BINARY_DIR@/test/conf"
 
-#if (_MSC_VER >= 1400) // Visual Studio 2005
-  #include <sstream>
-
-  /// \brief setenv/unstenv are not present in Windows. Define them to make
-  /// the code portable.
-  /// \param[in] _name Variable name.
-  /// \param[in] _value Value.
-  /// \param[in] _rewrite If 'name' does exist in the environment, then its
-  /// value is changed to 'value' if 'rewrite' is nonzero. If overwrite is
-  /// zero, then the value of 'name' is not changed.
-  /// /return 0 on success or -1 on error.
-  int setenv(const char *_name, const char *_value, int /*_rewrite*/)
-  {
-    std::stringstream sstr;
-    std::string name = _name;
-    std::string value = _value;
-    sstr << name << '=' << value;
-    return _putenv(sstr.str().c_str());
-  }
-
-  /// \brief Deletes an environment variable.
-  /// \param[in] _name Variable name.
-  void unsetenv(const char *_name)
-  {
-    std::stringstream sstr;
-    std::string name = _name;
-    sstr << name << '=';
-    _putenv(sstr.str().c_str());
-  }
+#ifdef _MSC_VER
+#    define popen _popen
```

Comment:
I know this is a backport, but are these lines related with these changes?

Reply:
Good point, the removal of the custom `setenv` functions goes well with the rest of the changes, but the `popen` defines are not directly related.

Since this was in the original PR, I'm leaning towards leaving it here to decrease the diff between branches. It shouldn't really affect `ign-gazebo3` because it's not supported on Windows.